README.md

```diff
@@ -34,6 +34,20 @@ Please follow our contribution guide [here](https://docs.bucketeer.io/contributi
 
 ## Development
 
+### Setup the library management
+This　project use [mint](https://github.com/yonaskolb/Mint) for library management.
+
+#### Install
+```sh
+make install-mint
+```
+※You need [homebrew](https://brew.sh/) to install mint.
+
+#### Install library
+```sh
+make bootstrap-mint
+```
+
```

Comment:
Hi @kakcy , I tried your update.
I am facing one problem when run `make bootstrap-mint`

```shell
make bootstrap-mint
./hack/mint.sh --bootstrap
./hack/mint.sh: line 18: [result_search_mint=]: command not found
 ^Cmake: *** [bootstrap-mint] Interrupt: 2
```

```shell
which mint
/opt/homebrew/bin/mint
```
 
 `mint` has been install successfully, and I could run `mint`.
 
 Please check my screenshot.
 
 
<img width="837" alt="Screenshot 2023-11-11 at 09 59 09" src="https://github.com/bucketeer-io/ios-client-sdk/assets/2597710/c2dada07-886a-478e-8714-c2e567cd6a53">

<img width="697" alt="Screenshot 2023-11-11 at 10 04 05" src="https://github.com/bucketeer-io/ios-client-sdk/assets/2597710/c83d1cd3-dc9c-4ea5-9937-32e085ab45ba">

Reply:
Hi @duyhungtnn, I'm sorry,but a coding mistake resulted in unintended behavior.
fix at https://github.com/bucketeer-io/ios-client-sdk/pull/50/commits/19dad4c213984c84fd3c7e2895f5feae4560687c.

Reply:
@kakcy I tried again, it seems to work and look good. 

<img width="1512" alt="Screenshot 2023-11-14 at 10 51 34" src="https://github.com/bucketeer-io/ios-client-sdk/assets/2597710/035b1987-8508-46e0-91cc-73e67956faa2">